packages/node-provider/PROTOCOL_SPEC.md

```diff
@@ -0,0 +1,69 @@
+# Protocol Spec
+
+## Public JSON-RPC Methods
+
+| method         | params                                                       | result                                                                  |
+|----------------|--------------------------------------------------------------|-------------------------------------------------------------------------|
+| cf_getApps     | []                                                           | {id, definition, state, asset, myDeposit, peerDeposit, timeout}[]       |
+| cf_install     | [definition, state, asset, myDeposit, peerDeposit, timeout]  | {id}                                                                    |
```

Comment:
What about proposeInstall?

Reply:
Based on https://github.com/counterfactual/monorepo/blob/master/packages/cf.js/API_REFERENCE.md

Seems like `proposeInstall` would return the ID of the AppInstance, which would _then_ be used to make the actual install upon acceptance, correct?